sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobEventStoreEventSource.cs

```diff
@@ -321,13 +321,13 @@ protected BlobEventStoreEventSource() : base(EventSourceName)
                                                   string consumerGroup,
                                                   string clientIdentifier,
                                                   string errorMessage,
-                                                  long sequenceNumber,
+                                                  string sequenceNumber,
                                                   int replicationSegment,
```

Comment:
Should we just make `replicationSegment` a string as well?   We know that it's basically a dummy value when there's no GeoDR enabled.   I don't feel strongly about it, but thought it worth mentioning.

Reply:
updated - I had them all as string initially but I don't remember why I changed it